src/mainviews/detail-view/job-detail.tsx

```diff
@@ -65,6 +65,33 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
 
   const ss = new SchedulerService({});
 
+  const translateStatus = useCallback(
+    (status: Scheduler.Status) => {
```

Comment:
```suggestion
    (status: Scheduler.Status | null | undefined) => {
```

With a `default:` block below, this would translate nullish statuses as `''`

Reply:
Understood, but status is never empty on DetailJob.